interfaces/builtin/pulseaudio.go

```diff
@@ -23,27 +23,157 @@ import (
 	"github.com/snapcore/snapd/interfaces"
 )
 
-const pulseaudioConnectedPlugAppArmor = `
-/etc/pulse/ r,
-/etc/pulse/* r,
+var pulseaudioConnectedPlugAppArmor = []byte(`
+capability dac_override,
+
+/etc/machine-id r,
```

Comment:
This is currently allowed in 'abstractions/dbus-session-strict'. On touch we specifically disabled the DBus socket and only allowed apps to connect via the native socket. Is this changing or is pulseaudio using /etc/machine-id for something unrelated?